tests/functional/amdApp/circular1.ts

```diff
@@ -0,0 +1,9 @@
+import * as circular2 from './circular2';
```

Comment:
Change this line to:
`import circular2 from './circular2';

Then below replace `circular2.default()` with `circular2()`.